packages/p2p/src/components/my-ads/toggle-ads.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes            from 'prop-types';
+import { localize }         from 'deriv-translations';
+import { ToggleSwitch }     from 'deriv-components';
+import classNames           from 'classnames';
+import './my-ads.scss';
+
+const ToggleMessage = props => {
```

Comment:
nit can we destructure the props here? to be consistent with our other functional components